bin/cli.js

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '--run-todo',                'Run todo tests'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+  [ '--karma-config <value>',    'Karma config file'],
+];
+
+const karmaLogLevels = {
+  'disable': [ karma.constants.LOG_DISABLE, 'none' ],
+  'error':   [ karma.constants.LOG_ERROR, 'errors-only' ],
+  'warn':    [ karma.constants.LOG_WARN, 'minimal' ],
+  'info':    [ karma.constants.LOG_INFO, 'normal' ],
+  'debug':   [ karma.constants.LOG_DEBUG, 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
```

Comment:
I think we have discussed it but, what's the difference between `'quiet'` and `'disable'`?